samples/nextjs/src/pages/_error.tsx

```diff
@@ -6,17 +6,20 @@ interface ErrorPageProps {
 }
```

Comment:
I see your comment in description:
'Refresh custom _error component - make it clear this covers both server and client vs the custom 500 page which only covers server.'
But previously it handled both client and server side errors, as I see. What was changed?

Reply:
Nothing really changed with [_error.tsx](https://nextjs.org/docs/advanced-features/custom-error-page#more-advanced-error-page-customizing) (correct, always handled both server and client). What changed is that Next has since also introduced the option to add a [custom 500 page](https://nextjs.org/docs/advanced-features/custom-error-page#customizing-the-500-page) (500.tsx), but this only handles server errors. I wanted to make it clear that our sample uses the more advanced option.